src/custom/pages/Claim/ClaimSummary.tsx

```diff
@@ -53,7 +69,7 @@ export function ClaimSummaryView({ showClaimText, totalAvailableText, totalAvail
           </Trans>
         </ClaimSummaryTitle>
       )}
-      {totalAvailableAmount && (
+      {totalAvailableAmount && activeClaimAccount && (
```

Comment:
I don't think you need this here, since the amount is only used as a flag.

You can control this in the parent component, as you already do.
I think it'll work the same without this extra parameter. Doesn't it?

Reply:
I think we need this also because the `totalAvailableAmount` is now CurrencyAmount object by default and this happens otherwise
![Screenshot from 2022-02-17 11-46-24](https://user-images.githubusercontent.com/34926005/154460634-c12fc57e-1fa3-4b41-90c9-f9b35df4f5e9.png)

Reply:
Alright then